src/ensembl/src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState.ts

```diff
@@ -21,7 +21,7 @@ import { Homeologue } from '../../types/homeologue';
 import { Publication } from '../../types/publication';
 import { Source } from '../../types/source';
 import { Metadata } from '../../types/metadata';
-import { CrossReference } from '../../types/crossReference';
+import { ExternalReference } from '../../types/externalReference';
```

Comment:
Are we alright with using relative paths for imports that are two levels above?

Reply:
🤷‍♂️ 

It's kind of a borderline case. I personally would use absolute imports, but they were written as relative here, so I decided to leave them as they are.